### PR TITLE
minor edits to running-migrations.md

### PR DESCRIPTION
**Pull request by ooknosi**
Added Truffle Develop as a local migration testing alternative to Ganache.
Clarified default creation of 1_initial_migration.js

### CODE REVIEW

File: src/docs/truffle/getting-started/running-migrations.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To run your migrations, run the following:
...
+This will run all migrations located within your project's `migrations` directory. At their simplest, migrations are simply a set of managed deployment scripts. If your migrations were previously run successfully, `truffle migrate` will start execution from the last migration that was run, running only newly created migrations. If no new migrations exists, `truffle migrate` won't perform any action at all. You can use the `--reset` option to run all your migrations from the beginning. For local testing, make sure to have a test blockchain such as [Ganache](/ganache) or Truffle Develop configured and running before executing `migrate`.
```
Comment (eggplantzzz) [inline src/docs/truffle/getting-started/running-migrations.md]: How do you feel about making the last line something like the following:
```
...make sure to have a test blockchain such as Ganache configured and running before executing `truffle migrate`. You can also use `truffle develop` and run your migrations.
```

File: src/docs/truffle/getting-started/running-migrations.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ contract Migrations {
 }
 ```
 
-You must deploy this contract inside your first migration in order to take advantage of the Migrations feature. To do so, create the following migration:
+You must deploy this contract inside your first migration in order to take advantage of the Migrations feature. To do so, the following migration is provided by default when creating a new project with `truffle init`:
```
Comment (eggplantzzz) [inline src/docs/truffle/getting-started/running-migrations.md]: What about, 
```
...to take advantage of the migrations feature. The following migration is provided by default when creating a new project with `truffle init`:
```
How do you feel about that wording?

**Comment by eggplantzzz** [comment]
Thanks for the PR @ooknosi and sorry for such delay in getting back to you. I made a couple of comments. I suppose we should probably also adjust the migrations file in this same file. It has a very old pragma.

**Comment by DiscRiskandBisque** [comment]
@ooknosi just pinging to follow-up--were able to see @eggplantzzz edits?

@eggplantzzz is this good to merge without your edits being addressed?

**Comment by eggplantzzz** [comment]
@DiscRiskandBisque Let's give him a week and if no response then I'll go ahead and edit and resolve the conflict.

**Comment by eggplantzzz** [comment]
@DiscRiskandBisque This PR should be good to merge now.

**Pull request merged**